### PR TITLE
Add queueLengthStrategy option to azure-storage-queue scaler.

### PR DESCRIPTION
**Pull request by leodip**
This adds documentation about the new `queueLengthStrategy` configuration option for the Azure Storage Queue scaler.

Related to https://github.com/kedacore/keda/issues/4478

### CODE REVIEW

File: content/docs/2.15/scalers/azure-storage-queue.md [hunk 1]
```diff
@@ -26,6 +27,7 @@ triggers:
 
 - `queueName` - Name of the queue.
 - `queueLength` - Target value for queue length passed to the scaler. Example: if one pod can handle 10 messages, set the queue length target to 10. If the actual number of messages in the queue is 30, the scaler scales to 3 pods. (Default: `5`, Optional)
+- `queueLengthStrategy` - `default` considers both visible and invisible messages, while `visibleonly` uses Peek to count only visible messages. In `visibleonly`, if the count of messages is 32 or higher, it falls back to the default strategy, counting both visible and invisible messages.
```
Comment (tomkerkhove) [inline content/docs/2.15/scalers/azure-storage-queue.md:31]: Default is not really clarifying what it includes. Instead I would propose to call it `all`

Comment (tomkerkhove) [reply content/docs/2.15/scalers/azure-storage-queue.md:31]: Also, please align with the "(Default: `5`, Optional)" convention as per https://github.com/kedacore/keda-docs#writing-documentation-for-a-scaler

**Comment by tomkerkhove** [comment]
Can you fix the DCO issues please? You can learn more in our [contribution guide](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#i-didnt-sign-my-commit-now-what).

File: content/docs/2.15/scalers/azure-storage-queue.md (hunk 1 above)
Comment (leodip) [reply content/docs/2.15/scalers/azure-storage-queue.md]: Thanks for the feedback. Hope this version is better?

**Comment by leodip** [comment]
DCO fixed.

**Review by tomkerkhove — APPROVED** [review]
LGTM, waiting for feature to merge first

**Review by zroubalik — APPROVED** [review]

**Pull request merged**